deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml

```diff
@@ -63,8 +63,43 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: egov-idgen
-  - name: EGOV_WORKFLOW_HOST
-    valueFrom:
-      configMapKeyRef:
-        name: egov-service-host
-        key: egov-workflow-v2
+  - name: ATTENDANCE_SERVICE_LOG_KAFKA_CREATE_TOPIC
+    value: "save-attendance-log"
+  - name: ATTENDANCE_SERVICE_LOG_KAFKA_UPDATE_TOPIC
+    value: "update-attendance-log"
+  - name: ATTENDANCE_SERVICE_REGISTER_KAFKA_CREATE_TOPIC
```

Comment:
Remove the service bit from ATTENDANCE_SERVICE_LOG_KAFKA**. Service is understood. Can be just ATTENDANCE_LOG**, ATTENDANCE_REGISTER**.

Reply:
Word service has been removed